runtime/stdlib/contracts/test.cdc

```diff
@@ -134,6 +134,27 @@ pub contract Test {
         pub fun moveTime(by delta: Fix64) {
             self.backend.moveTime(by: delta)
         }
+
+        /// Creates a snapshot of the blockchain, at the
+        /// current ledger state, with the given name.
+        ///
+        pub fun createSnapshot(_ name: String) {
```

Comment:
Let's use `access(all)` instead of `pub` keyword for any newly added Cadence code since it is removed in stable cadence. This will make it easier when keeping the stable-cadence branch up-to-date.

```suggestion
        access(all) fun createSnapshot(_ name: String) {
```

Reply:
Good idea :ok_hand: 
Fixed in https://github.com/onflow/cadence/pull/2763/commits/0c10225154a7124956a534e6cc484850989c615a